Sources/libhostmgr/Platforms/LocalVMImage.swift

```diff
@@ -52,7 +56,8 @@ public struct LocalVMImage: Equatable {
     }
 }
 
-public enum VMImageState: String {
-    case packaged = "Packaged"
-    case ready = "Ready"
+public enum VMImageState: String, CaseIterable {
+    case packaged = "📦 Packaged"
+    case ready = "💾 Ready"
+    case running = "▶️ Running"
```

Comment:
```suggestion
    case running = "🟢 Running"
```

Nit: The icon you specified renders ASCII-like in the terminal, which causes weird spacing issues in the table.

Not a blocker.